src/devices/kangtai.c

```diff
@@ -0,0 +1,77 @@
+#include "rtl_433.h"
```

Comment:
Add a proper copyright header and preferably a tiny description of the protocol. Look in other files how it is done.